index.js

```diff
@@ -54,15 +54,15 @@ function mapOutput(stdout, filter, mapping, coefficient = 1) {
 /**
  * Run the command and do common things between win32 and unix
  *
- * @param {String} cmd - The command to execute
+ * @param {String} cmd - The command to execute (must be in list/array)
```

Comment:
You should do
`@param {String[]} cmd`
for a proper JSDoc.

Reply:
Thanks, I don't use JSDoc!

I also applied the changes you requested.